### PR TITLE
Add note for aws china endpoint

### PR DESCRIPTION
**Pull request by abhi18av**
This PR adds the configuration for AWS China regions in the docs, a follow up from https://github.com/nextflow-io/nextflow/issues/1952#issuecomment-961276439

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -607,6 +607,7 @@ For example::
         }
     }
 
+.. tip:: To use an AWS China region, please specify the appropriate region code in ``endpoint``. For example ``aws.client.endpoint = "s3.cn-north-1.amazonaws.com.cn"``
```
Comment (pditommaso) [inline docs/config.rst:610]: Does not the endpoint start with `https://`?

Comment (abhi18av) [reply docs/config.rst:610]: On the comment which confirmed this setting, there's no `https://` - but yeah, I agree we should be more explicit on our end.

Comment (abhi18av) [reply docs/config.rst:610]:
```suggestion
.. tip:: To use an AWS China region, please specify the appropriate region code in ``endpoint``. For example ``aws.client.endpoint = "https://s3.cn-north-1.amazonaws.com.cn"``
```

Comment (pditommaso) [reply docs/config.rst]: Now that you are mentioning it, I'm not sure anymore. Actually, in the docs there are no protocol prefix. 

https://docs.aws.amazon.com/general/latest/gr/s3.html#s3_website_region_endpoints

This we should to a test to verify it

Comment (abhi18av) [reply docs/config.rst]: Regarding this, I was able to test access with `https`. Also, this is supported by 


- The [virtual hosting](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) docs.

- The s3 endpoint docs

**Comment by pditommaso** [comment]
Ok, I've moved to S3 docs page and merged manually 9c8110406. tx!

**Pull request closed without merging**